ihmc-footstep-planning/src/main/java/us/ihmc/footstepPlanning/log/FootstepPlannerLogger.java

```diff
@@ -49,21 +50,23 @@ public class FootstepPlannerLogger
    private static final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyyMMdd_HHmmssSSS");
    private static final int RECOMMENDED_NUMBER_OF_LOGS_TO_KEEP = 500;
```

Comment:
Not super important but RECOMMENDED_NUMBER_OF_LOGS_TO_KEEP could be swapped for a variable that keeps the last N days of logs instead, since things are in terms of dates now.